test/integration_test.rb

```diff
@@ -78,7 +78,7 @@ def test_semantic_highlighting
     assert_telemetry("textDocument/didOpen")
 
     response = make_request("textDocument/semanticTokens/full", { textDocument: { uri: "file://#{__FILE__}" } })
-    assert_empty(response[:result][:data])
+    assert_equal([0, 6, 3, 2, 0], response[:result][:data])
```

Comment:
We should reconsider how we test this (in another PR) because this is surprisingly cryptic.